File: C++/1010.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int main()
{
    int code1, code2, units1, units2;
    double price1, price2, total;

    cin >> code1 >> units1 >> price1;
    cin >> code2 >> units2 >> price2;

    total = units1 * price1 + units2 * price2;

    cout << "VALOR A PAGAR: R$ " << fixed << setprecision(2) << total << endl;
    return 0;
}

File: Python/1010.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

code_1, units_1, price_1 = input().split()
code_2, units_2, price_2 = input().split()

total = int(units_1) * float(price_1) + int(units_2) * float(price_2)

print(f"VALOR A PAGAR: R$ {total:.2f}")
